### PR TITLE
[SGDv2] Add Tune-Cifar-PyTorch-PBT example

### PR DESCRIPTION
**Pull request by Yard1**
## Why are these changes needed?

Adds an adapted SGDv1 example for SGDv2 & Tune.

The conversion was trivial, and I have not ran into any issues. The new API is intuitive and streamlined. I got it to work on the first try, without any issues.

## Checks

- [x] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: python/ray/util/sgd/v2/BUILD [hunk 1]
```diff
@@ -34,6 +44,7 @@ py_test(
     args = ["--smoke-test"]
 )
 
+
```
Comment (Yard1) [inline python/ray/util/sgd/v2/BUILD:47]:
```suggestion

```

**Review by amogkam — CHANGES_REQUESTED** [review]
Thanks a lot @Yard1! It looks like the example is failing on CI though. Could you also add this example to the docs: https://docs.ray.io/en/master/raysgd/v2/examples.html

**Comment by Yard1** [comment]
@amogkam My bad! Updated the example. Also enabled it to work for GPU, please check if anything else is required!

**Review by amogkam — APPROVED** [review]
Thanks!

**Pull request merged**